experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -749,29 +755,22 @@ export class ModularChangeFamily
 	}
 }
 
-/**
- * @alpha
- */
-export function revisionMetadataSourceFromInfo(
+export function rebaseRevisionMetadataFromInfo(
 	revInfos: readonly RevisionInfo[],
-): RevisionMetadataSource {
-	const getIndex = (revision: RevisionTag): number | undefined => {
-		const index = revInfos.findIndex((revInfo) => revInfo.revision === revision);
-		return index >= 0 ? index : undefined;
-	};
-	const tryGetInfo = (revision: RevisionTag | undefined): RevisionInfo | undefined => {
-		if (revision === undefined) {
-			return undefined;
+	baseRevisions: (RevisionTag | undefined)[],
```

Comment:
Please add doc comments so it's clear whether `revInfos` should contain the `RevisionInfo` for the base and rebased changes or just the rebased changes.